source/standards/hosting.html.md.erb

```diff
@@ -1,71 +1,54 @@
 ---
-title: Hosting
+title: How to host services
 expires: 2018-05-01
 ---
 
 # <%= current_page.data.title %>
 
 <%= partial :expires %>
 
-GDS follows the [Government Cloud First policy](https://www.gov.uk/guidance/government-cloud-first-policy) meaning that we prefer the use of Platform as a service (PaaS) and Infrastructure as a Service (IaaS) solutions over managing our own hardware.
+GDS follows the [Government Cloud First policy](https://www.gov.uk/guidance/government-cloud-first-policy) preferring Platform as a service (PaaS) and Infrastructure as a Service (IaaS) solutions over managing our own hardware.
 
-## User Needs
-Your hosting solution will need to accommodate the user needs of:
-* service users who want systems that are highly scalable and available
-* GDS administrators who want to manage their environments easily, simply and with confidence using automated tools
+Your team can build new tools on a primary cloud platform, we prefer:
 
-## Principles
+* systems built on a PaaS instead of on IaaS
+* cloud suppliers which are well supported by common tools
 
-GDS will build new tools on a primary cloud platform for each of PaaS and IaaS.
+There is some useful information in the [Service Manual](https://www.gov.uk/service-manual/technology/deciding-how-to-host-your-service) about deciding how to host your service.
 
-* we prefer systems built on a PaaS to systems built on IaaS where possible
-* we prefer cloud suppliers that are well supported by common tooling to reduce cost of migrating to a new supplier
+## Choose your hosting solution
 
-## Tools
+Your hosting solution should accommodate:
 
-GDS has selected two cloud platforms:
+* service users who want highly scalable and available systems
+* GDS administrators who want to manage their environments simply using automated tools
 
-### PaaS
+GDS has decided to use following cloud platforms:
 
-GOV.UK Platform as a Service (PaaS) manages the deployment of your apps and services.
-* GOV.UK PaaS
-  https://cloud.service.gov.uk
+* [GOV.UK Platform as a Service (PaaS)](https://cloud.service.gov.uk) which manages the deployment of your apps and services
+* [Amazon Web Services (AWS)](https://aws.amazon.com) which offers scalable computing, storage and deployment services
 
-### IaaS
+## Consider vendor switching costs
 
-Amazon Web Services (AWS) offer scalable computing, storage and deployment services.
-* Amazon Web Services
-  https://aws.amazon.com
+AWS has a large number of available services. Some are common to many providers, such as compute and file storage, others are more specific to AWS itself. The services you choose may impact switching costs both financially and in the time taken to switch.
```

Comment:
Do you mean the cost of switching to a different vendor in the future? If so, can you highlight that ("future switching costs" perhaps)?